src/features/reasoning/helpers.ts

```diff
@@ -40,7 +40,7 @@ export const estimateTokenUsage = (
   };
 };
 
-export async function calculateDeepseekTokenUsage(
+export async function calculateFinalChunkTokenUsage(
```

Comment:
We can use this function to track qwen token usage by adding `stream_options: { include_usage: true }`.

That does impact how we parse the stream, since we will now have a final chunk with no content choices, but with a usage object. We need to account for that as a content chunk and not route it as a tool call as is the current schema.